src/data/clients.ts

```diff
@@ -637,6 +637,30 @@ export const Clients: Array<Client> = [
     ],
     recommended: true
   },
+  {
+    id: 'phyn',
+    name: 'Phyn',
+    description: 'Multiplatform video and audio playback client for Jellyfin with Jellyseerr support. Available on iOS, iPadOS and macOS. Android coming soon...',
```

Comment:
- We do not advertise piracy tools (Jellyseerr) from out documentation
- The "Android coming soon" part is not worth mentioning here

Reply:
thanks for the feedback. I have fixed it